### ISSUE TITLE
Add sort and filter to data fields in user dashboard

### ISSUE THREAD

**Issue by anik-ghosh-au7** [authorizerdev/authorizer#112]
**Feature Description**
- add sort and filter options to data fields in user dashboard

**Describe the solution you'd like**
- add sort to `created_at` field
- add filter to `email_verified` field

**Describe alternatives you've considered**
- Not applicable

**Additional context**
- Not applicable

**Comment by Shofiya2003** [comment]
Hey there!
I went through the documentation. I am not familiar with GraphQL APIs yet. We do not need to add sort and filter to the API to work on this issue. Right? I need to make changes in [this repository](https://github.com/authorizerdev/authorizer/tree/main/dashboard)
Can I please work on this issue? It would take me some time to get familiar with the architecture.
thanks a ton

**Comment by lakhansamani** [comment]
@Shofiya2003 sure you can work on this issue. You will have to add sort and filter to various database queries that we support. So you might need to know or understand querying structure for databases that we support.

Thanks

Happy to help if you need any architectural help.

**Comment by anik-ghosh-au7** [comment]
yes @Shofiya2003, you are correct that GraphQL APIs can provide sorting and filtering options out of the box using the `where` and `filter` clauses in the query from the frontend
